README.md

```diff
@@ -153,6 +153,18 @@ KUBEMONKEY_TIME_ZONE=America/New_York
 enabled= true
 schedule_immediate_kill= true
 ```
+#### Example Config for posting attack notifications to an HTTP endpoint
+```toml
+[notifications]
+  enabled = true
+  [[notifications.attacks]]		 
+    endpoint = "http://url1"
+    message = "message1"
+    headers = ["header1Key:header1Value","header2Key:header2/Value"]
```

Comment:
It would be great to have a default message and headers (possibly in json format?).

Reply:
Imho default values for message/headers wouldn't add value + might cause issues.
I'd rather want to see Kubemonkey exit instantly if (endpoint, message, headers) are not there in the configs than moving forward with default values.